rllib/algorithms/algorithm_config.py

```diff
@@ -3227,7 +3240,11 @@ def get_learner_group_config(self, module_spec: ModuleSpec) -> LearnerGroupConfi
             )
             .resources(
                 num_learner_workers=self.num_learner_workers,
-                num_cpus_per_learner_worker=self.num_cpus_per_learner_worker,
+                num_cpus_per_learner_worker=(
+                    self.num_cpus_per_learner_worker
+                    if not self.num_gpus_per_learner_worker
+                    else 0
+                ),
```

Comment:
qq: this is not needed right? cause `self.num_cpus_per_learner_worker` will be zero if `self.num_gpus_per_learner_worker > 0`.  If this is not the case, we already raise errors.

Reply:
yeah I think I just added this in here before I added the error. We don't technically need this Ill remove it if I need to do anything to get the release tests to run.